index.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
-var objectAssign = require('object-assign');
-var sep = {};
+const sep = {};
 
-function create(input, opts) {
-	return input.map(function (line) {
+const create = (input, opts, menuLevel = 0) => {
+	return input.map(line => {
```

Comment:
I believe this is my last comment to this PR :p. But I might be more in favour of something like this

```js
const create = (input, opts, menuLevel = 0) => input.map(line => {

});
```

or


```js
const create = (input, opts, menuLevel = 0) => 
    input.map(line => {

    });
```

Up to @sindresorhus to decide what he prefers.